src/Razor/src/Microsoft.VisualStudio.RazorExtension/AssemblyBindingRedirects.cs

```diff
@@ -43,26 +43,26 @@
     GenerateCodeBase = true,
```

Comment:
Why did this not change the binding redirects for `Microsoft.Extensions.ObjectPool`? The MSBuild property that you changed controls that reference too.

Reply:
@jaredpar: You might know this best. Will it affect source build if we update Microsoft.ExtensionsObjectPool to 7.0.0.0?

Reply:
Responded to the thread to try and explain my mental check list for changes like this. 😄

Reply:
Yup, just saw the email -- thanks!

Reply:
> Why did this not change the binding redirects for Microsoft.Extensions.ObjectPool? The MSBuild property that you changed controls that reference too.

Originally also changed it for `Microsoft.Extensions.ObjectPool`. Reverted it back for ObjectPool to get a fix for the following error:

> CreatePkgDef : error : ProvideBindingRedirectionAttribute: Invalid value specified for NewVersion.

Reply:
ObjectPool is versioned separately because of SourceBuild: https://github.com/dotnet/razor/pull/8657

Reply:
ohhh! good to know